docs/overview/reference/waku-vs-libp2p.md

```diff
@@ -1,22 +1,21 @@
 ---
-title: Waku vs libp2p - what's the difference?
+title: Waku vs libp2p - What's the difference?
```

Comment:
I added a "Reference" section specifically for important guides not related to user onboarding for Waku but are still crucial for users to be aware of and refer to. Since Waku is built on libp2p, understanding their differences is important, even though it doesn't directly involve onboarding users to Waku. The purpose of the reference section is to house these guides, along with other resources like a glossary and FAQ (when available).